src/components/Card/Card.vue

```diff
@@ -1,28 +1,43 @@
 <template>
   <article class="location-card" :id="id">
-    <div class="button-container">
-      <button class="button button-refresh" type="button" name="button" v-on:click.prevent="refreshLocation">🔄</button>
-      <button class="button" type="button" name="button" v-on:click.prevent="deleteCard">X</button>
+    <div v-if="currentLocation">
+      <p>Current Location</p>
     </div>
-    <h2>{{ locale }}</h2>
+    <div v-else class="button-delete-container">
+      <button class="card-button button-delete" type="button" name="button" v-on:click.prevent="deleteCard">X</button>
+    </div>
+    <h2>{{ formatLocation() }}</h2>
     <img :src="require(`../../assets/${formatWeatherIcon()}.png`)" :alt="weatherIconAlt" class="weather-icon">
     <h3>Weather</h3>
     <p>{{ temperature }}°F</p>
     <p>Wind Speed: {{ windSpeed }}mph</p>
     <p>Humidity: {{ humidity }}%</p>
+    <hr>
     <h3>Air Quality</h3>
-    <p>{{ aqi }} AQI</p>
-    <p>{{ determineAqiMessage() }}</p>
-    <p>{{ timeStamp }}</p>
+    <div class="aqi-message">
+      <div class="aqi-circle" :style="determineAqiStyle()"><p>AQI: {{ aqi }}</p></div>
+      <p class="aqi-rating"><b>{{ determineAqiRating() }} </b><a href="https://www.airnow.gov/aqi-and-health/" target="_blank"><img src="https://img.icons8.com/metro/16/000000/info.png"/></a></p>
+      <p class="aqi-text">{{ determineAqiText() }}</p>
+    </div>
+    <hr>
+    <button class="button-allTrails" type="button" name="button">
+      <a :href="generateAllTrailsURL()" target="_blank" class="button-allTrails-text">Plan an activity in {{ city }} on AllTrails</a>
```

Comment:
Nice job using the target property to open a new window! I've seen this in examples but haven't ever implemented it and this seems like the perfect place to do so.